config.example/group_vars/all.yml

```diff
@@ -3,13 +3,6 @@
 
 # Account to use for SSH connections
 ansible_user: ubuntu
- 
-# SSH Configuration
-ssh_password_authentication: 'yes'
-
-# Configure SUDO to not require a password for users in the `sudo` group
-# If this is `false`, pass the `-K` flag to all Ansible commands to prompt for the SUDO password
-enable_sudo_without_password: true
```

Comment:
I think you just added this, and enabling sudo without a password is something that's done by default, so if you want it to be an option then we'll need this one.

Reply:
It’s usually done in the bootstrap playbook